Randomize.py

```diff
@@ -169,5 +183,22 @@ def modymlcopy(old,new=False,platform=False,internal=False):
     elif randomresult[i] == 3:
         modymlcopy('W_EX010_KH1F','W_EX010_KH1F_MAGIC')
 
+h.write('- name: 00battle.bin\n')
+h.write('  method: binarc\n')
+h.write('  source:\n')
+h.write('  - name: ptya\n') #Ragnarok Fix
+h.write('    type: list\n')
+h.write('    method: copy\n')
+h.write('    source:\n')
+h.write('    - name: ptya'+forms[newlimit]+'.list\n')
+#Base Sora's active abilities (still may edit 00battle.bin)
+if randomresult[0] != 0:
+    h.write('  - name: plrp\n') 
+    h.write('    type: list\n')
+    h.write('    method: copy\n')
+    h.write('    source:\n')
+    h.write('    - name: plrp.yml\n')
```

Comment:
Change to below as current writes and invalid mod.yml crashing the game. Tested with all form and suggested code works fine
```
h.write('- name: 00battle.bin\n')
h.write('  method: binarc\n')
h.write('  source:\n')
h.write('  - name: ptya\n') #Ragnarok Fix
h.write('    method: copy\n')
h.write('    source:\n')
h.write('    - name: ptya'+forms[newlimit]+'.list\n')
h.write('    type: list\n')
#Base Sora's active abilities (still may edit 00battle.bin)
if randomresult[0] != 0:
    h.write('  - name: plrp\n') 
    h.write('    method: listpatch\n')
    h.write('    type: List \n')
    h.write('    source:\n')
    h.write('    - name: plrp.yml\n')
    h.write('      type: plrp\n')
    ```